packages/trader/src/Modules/Reports/Containers/reports.jsx

```diff
@@ -89,20 +91,24 @@ class Reports extends React.Component {
 }
 
 Reports.propTypes = {
-    disableRouteMode: PropTypes.func,
-    enableRouteMode : PropTypes.func,
-    history         : PropTypes.object,
-    is_visible      : PropTypes.bool,
-    location        : PropTypes.object,
-    routes          : PropTypes.arrayOf(PropTypes.object),
-    toggleReports   : PropTypes.func,
+    disableRouteMode  : PropTypes.func,
+    enableRouteMode   : PropTypes.func,
+    history           : PropTypes.object,
+    is_visible        : PropTypes.bool,
+    location          : PropTypes.object,
+    routes            : PropTypes.arrayOf(PropTypes.object),
+    setModalIndex     : PropTypes.func,
+    toggleReports     : PropTypes.func,
+    vertical_tab_index: PropTypes.number,
 };
 
 export default connect(
     ({ ui }) => ({
-        disableRouteMode: ui.disableRouteModal,
-        enableRouteMode : ui.setRouteModal,
-        is_visible      : ui.is_reports_visible,
-        toggleReports   : ui.toggleReports,
+        disableRouteMode  : ui.disableRouteModal,
+        enableRouteMode   : ui.setRouteModal,
+        is_visible        : ui.is_reports_visible,
+        setModalIndex     : ui.setModalIndex,
```

Comment:
`setModalIndex` was changed to `setVerticalTabIndex` in UI store; let's update it here?